test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -4,14 +4,55 @@
 
 
 class AccountConnectionWizard:
-    SERVER_ADDRESS_BOX = names.leUrl_OCC_PostfixLineEdit
-    NEXT_BUTTON = names.owncloudWizard_qt_passive_wizardbutton1_QPushButton
-    USERNAME_BOX = names.leUsername_QLineEdit
-    PASSWORD_BOX = names.lePassword_QLineEdit
-    SELECT_LOCAL_FOLDER = names.pbSelectLocalFolder_QPushButton
-    DIRECTORY_NAME_BOX = names.fileNameEdit_QLineEdit
-    CHOOSE_BUTTON = names.qFileDialog_Choose_QPushButton
-    CONNECT_BUTTON = names.owncloudWizard_qt_passive_wizardbutton1_QPushButton
+    SERVER_ADDRESS_BOX = {
+        "buddy": names.owncloudWizard_label_2_QLabel,
+        "name": "leUrl",
+        "type": "OCC::PostfixLineEdit",
+        "visible": 1,
+    }
+    NEXT_BUTTON = {
+        "name": "__qt__passive_wizardbutton1",
+        "type": "QPushButton",
+        "visible": 1,
+        "window": names.owncloudWizard_OCC_OwncloudWizard,
+    }
+    USERNAME_BOX = {
```

Comment:
I think you can split these changes to a different PR  since this is not sharing related

Reply:
Fixed